### ISSUE TITLE
New espeak variants change the pitch.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#7152]
1. open voice settings when you have espeak selected.
2. Select French English.
3. Note that when arrowing up from French english, and back down below it, there's a weird pause in NVDA, (No speech is sounded). If I turn variant to None, it changes pitch (It talks really insanely high).

**Comment by bhavyashah** [comment]
@derekriemer I am unable to reproduce the described issue. Firstly, by Espeak, I take it that you are referring to the newer ESpeak-NG engine NVDA has incorporated, right? In any case, with ESpeak-NG selected in an installed copy of NVDA version 2017.2, I cannot find any language or item of the Voices combo box, particularly around French (written in English) which NVDA does not announce. I repeated this exercise with the None variant without fruit. I selected French and hit Ok, and repeated the aforesaid process with my favourite Steph 3 as well as None, yielding nothing akin to what you describe. Feel free to update this ticket by checking if this issue still persists at your end, is relevant to ESpeak-NG or in case you have a more specific STR to share.

**Comment by bhavyashah** [comment]
As noted in my previous comment, I am unable to replicate this issue. @derekriemer Does this bug still stand for you? If not, please close.

**Comment by derekriemer** [comment]
Hold off. That was reverted from next, it'll be in maybe again. cc @feerrenrut because I think he's doing the upgrade for that.

**Comment by feerrenrut** [comment]
Yes, there is an update to `espeak-ng` looming. It will be interesting to know if this bug shows up when it hits `next`. See PR #7385

**Comment by ehollig** [comment]
@derekriemer, now that #7385 is merged into Master, I am still unable to locate a voice titled "French English".

**Comment by Brian1Gaff** [comment]
What on earth would a French English sound like? Inspector Cleasou, no I 
can't spell it.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by Mary5958** [comment]
I guess it was more a Mrola voice; they had English with various accents like German, French and some others, and it was included in the temporary eSpeak update in May, but those Mbrola voices wouldn't work properly anyway, so now that they're not part of our updates anymore that voice is out.

**Comment by ehollig** [comment]
I do not believe I am experiencing this issue. @derekriemer and @feerrenrut can you confirm?

**Comment by feerrenrut** [comment]
I don't think I have ever been able to reproduced this. I don't know what "French English" means. I think it's best to close this issue for now. If @derekriemer or anyone can still reproduce this issue, please comment with instructions on how and we will re-open it.
